email2phonenumber.py

```diff
@@ -11,6 +10,12 @@
 import urllib
 from itertools import product
 from bs4 import BeautifulSoup
+import logging
+
+# Basic configuration for logging
+logging.basicConfig(format='%(asctime)s  %(levelname)-10s %(processName)s  %(name)s %(message)s')
```

Comment:
I don't think the additional information beyond the actual message is necessary unless you are debugging the tool. And it creates too much noise on the output.

Reply:
Solved 👍